AirCasting/ABConnector/SDCardAirBeamServices.swift

```diff
@@ -49,6 +49,8 @@ class BluetoothSDCardAirBeamServices: SDCardAirBeamServices {
         var receivedMeasurementsCount: [SDCardSessionType: Int] = [:]
         var currentSessionType: SDCardSessionType?
         
+        Log.info("[SD Sync] Downloading data")
```

Comment:
we can investigate a possibility to add a parameter to our logger for better type safety (with manual approach we can make a typo and miss logs):
`Log.info("Downloading data", subsystem: .sdSync)` or smth